src/scripts/argo_migration_workflow.sh

```
@@ -144,15 +145,77 @@ function await_and_apply_feedback() {
+    local phase="$1"
+    [[ -z "$phase" ]] && { echo -e "${RED}❌ No phase specified.${NC}"; return 1; }
+
+    _print_colored() {
+      local color="$1"
+      shift
+      echo -e "${color}$*${NC}"
+    }
+
+    case "$phase" in
+      "safe")
+        _print_colored "${YELLOW}" "$(cat <<EOF
```

Shellcheck (0.10.0)

[error] Couldn't parse this here document. Fix to allow more checks.

(SC1073)